dashboards-observability/public/components/application_analytics/helpers/utils.tsx

```diff
@@ -1,9 +1,14 @@
+/* eslint-disable no-console */
```

Comment:
Do we need this line? do you want to disable lint here?

Reply:
Yes because eslint does not allow any console logs and we use one in the fetchAppById function for when there is an error.